hazelcast-sql/pom.xml

```diff
@@ -319,6 +319,7 @@
                                         <exclude>META-INF/MANIFEST.MF</exclude>
                                         <exclude>codegen/**</exclude>
                                         <exclude>**/package-info.java</exclude>
+                                        <exclude>org/apache/calcite/runtime/CalciteResource.properties</exclude>
```

Comment:
This shows for me only in one jar, what's the second file you found?

Reply:
The `hazelcast-sql` module itself

https://github.com/hazelcast/hazelcast/blob/e657bb0ccacad274074bf28ebc905ea6998a1d97/hazelcast-sql/src/main/resources/org/apache/calcite/runtime/CalciteResource.properties#L1-L2